lib/bundle/brew_installer.rb

```diff
@@ -14,14 +14,16 @@ def initialize(name, options = {})
       @full_name = name
       @name = name.split("/").last
       @args = options.fetch(:args, []).map { |arg| "--#{arg}" }
+      @conflicts_with_arg = options.fetch(:conflicts_with, false)
```

Comment:
It seems to make more sense to:
```ruby
@conflicts_with_arg = options.fetch(:conflicts_with, [])
# OR
@conflicts_with_arg = options.fetch(:conflicts_with, Set.new)
```

Mixing boolean and array/set is kinda weird to me.